packages/remark-stringify/lib/defaults.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   tablePipeAlign: true,
   stringLength: stringLength,
   incrementListMarker: true,
+  definitionBlankLine: true,
```

Comment:
How about `tightDefinitions`? The `tight` prefix is used in several places in unified for options that use less whitespace.

Reply:
👍fdb5d919fdd5e1784176e85226459b6156e79902